WordPress.gitignore

```diff
@@ -11,3 +11,8 @@ wp-content/cache/
 wp-content/upgrade/
 wp-content/uploads/
 wp-content/wp-cache-config.php
+wp-content/plugins/hello.php
+
+/readme.html
+/license.txt
```

Comment:
I found this thread
http://wordpress.stackexchange.com/questions/60620/is-it-breaking-the-gpl-if-you-remove-license-txt-before-uploading